DNN Platform/Library/Common/Utilities/EscapedString.cs

```diff
@@ -74,7 +74,7 @@ public static IEnumerable<string> Seperate(string combinedString, char seperator
                 return result;
             }
 
-            var segments = combinedString.Split(new[] { seperator });
+            var segments = combinedString.RemoveNewLineSeparators().Split(new[] { seperator });
```

Comment:
Since this is a public method, I don't think we can alter this behavior without adding another argument to control whether or not to include whitespace.

Secondly, I think it would be more straightforward to call `Trim` on each segment, rather than do a `Regex.Replace` on the whole thing.  This would account for more whitespace characters than the hardcoded list, and would allow whitespace in the list values, just not around the values.  That seems like a better choice for a general-purpose utility method like this.

Reply:
@bdukes thanks for this valuable input. You have mentioned important point regarding whitespace characters in the list values. Initial idea was to remove all occurrences of whitespace characters, so I had to to apply Regex.Replace to an entire string  (i.e. there is `jpeg` but user accidentally put the line break and we got `jpe\ng`).

Do you think we should keep those characters inside and trim them around only?

Reply:
I agree with all your suggestions @bdukes Here is another solution.
Argument to trim whitespaces is added along with a multiple overloaded methods to ensure we do not introduce breaking changes into existing solution. Files extensions are trimmed now while other consumers of this `Separate` utility method works as before without changes.